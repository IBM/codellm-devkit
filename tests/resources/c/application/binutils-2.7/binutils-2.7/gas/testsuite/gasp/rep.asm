	.AREPEAT	5
	FIVE
	.AREPEAT	2
	TWO
	.AENDR
	.AREPEAT	3
	THREE
	.AREPEAT	2
	TWO	
	.AENDR
	.AENDR
	.AENDR
	.END
