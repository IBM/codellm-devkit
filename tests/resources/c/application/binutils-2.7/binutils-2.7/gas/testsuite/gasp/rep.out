!	.AREPEAT	5
!	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	.AENDR
!	FIVE
	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	1
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	2
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	4
!	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	.AENDR
!	FIVE
	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	1
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	2
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	3
!	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	.AENDR
!	FIVE
	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	1
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	2
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	2
!	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	.AENDR
!	FIVE
	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	1
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	2
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	.AENDR
!	FIVE
	FIVE
!	.AREPEAT	2
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	1
!	TWO
!	.AENDR
!	TWO
	TWO
!	.AREPEAT	3
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	2
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	.AENDR
!	THREE
	THREE
!	.AREPEAT	2
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.AREPEAT	1
!	TWO	
!	.AENDR
!	TWO	
	TWO	
!	.END
